src/predict.rs

```diff
@@ -38,7 +38,6 @@ pub static RAV1E_INTRA_MODES_MINIMAL: &'static [PredictionMode] = &[
 
 pub static RAV1E_INTER_MODES: &'static [PredictionMode] = &[
   PredictionMode::GLOBALMV,
-  PredictionMode::DC_PRED,
```

Comment:
Maybe could be made a separate commit to tell why.

Reply:
In this PR the list of modes tested in inter frames is constructed from the combination of a list of inter modes and a list of intra modes. There is thus no need anymore to list intra modes in the list of inter modes.